manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionTransformer.java

```diff
@@ -37,6 +43,8 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
     // once we use a single topic for every connector there will be no need of having a different
     // one per connector https://issues.redhat.com/browse/MGDSTRM-5977
     private String generateKafkaTopicName(String processorId) {
-        return topicName;
+        return rhoasService.isEnabled()
+                ? rhoasService.createTopicAndGrantAccessForProcessor(processorId, RhoasTopicAccessType.PRODUCER)
```

Comment:
I'm not sure I'd call the service during the transformation. Let's discuss it in person

Reply:
@lucamolteni according to our discussion, I moved the creation to `ConnectorsServiceImpl`.